en/config_vtol/vtol_quad_configuration.md

```diff
@@ -117,15 +115,15 @@ Make sure the VTOL is in a stable hover before you start the transition.
 Start your transition.
 It should transition within 50 – 100 meters.
 If it doesn’t or it isn’t flying in a stable fashion abort the transition (see below) and land or hover back to the start position and land.
-Try increasing the [transition throttle](#transition_throttle) (`VT_F_TRANS_THR`) value.
+Try increasing the [transition throttle](#transition-throttle) (`VT_F_TRANS_THR`) value.

```

Comment:
@sfuhrer FYI, you'll note I removed the anchor "#transition_throttle" from above the heading and changed the link to use a `-`. This is the default heading anchor format for vuepress and it works better for linking on vuepress than having an explicit anchor.

Generally nowdays I use the vuepress format for headings and link directly. I use the default heading anchor unless it is really long and likely to be linked in a lot of places. 

. The way we do link headings is a little arbitrary - in gitbook they used underscore separators and in vuepress headings get `-` separators by default.